vibe/src/main/java/com/vibeapp/vibe/models/ExtendedViewRepository.java

```diff
@@ -0,0 +1,9 @@
+package com.vibeapp.vibe.models;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+
+
+public interface ExtendedViewRepository extends JpaRepository<ExtendedView, Integer> {
+    ExtendedView findByuId(int uId);
```

Comment:
pretty sure this 'findBy' syntax isnt correct. IIRC its all camel case except 'find' - so that would mean findByUId... But this might also fail because of the double capital. The naming you and Allen did with uid also caused a lot of issues yesterday when i was trying to fix the build